### PR TITLE
fix: harvesting node link issue fixed

### LINKED ISSUE #1812: Can not request Harvesting
WTR:

1) Go to the Harvesting page
2) Click `Link the key` button
3) Click `Request Harvesting` button

Result: I can see two password popups instead of request harvesting
Video: https://www.loom.com/share/e03182b477dd4a8f8549e01ed803f2b2

### PR DESCRIPTION
**Pull request by yilmazbahadir**
fixes #1812 

* filtered the interested changes only
* set default fee to **slow** _instead of slowest_
* fixed refreshing harvesting status on signer change

### CODE REVIEW

**Review by cryptoBeliever — APPROVED** [review]

File: src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts [hunk 1]
```diff
@@ -149,7 +150,7 @@ export class FormPersistentDelegationRequestTransactionTs extends FormTransactio
     public formItems = {
         nodeModel: { nodePublicKey: '' } as NodeModel,
         signerAddress: '',
-        maxFee: 1,
+        maxFee: defaultFeesConfig.slow, // default: slow fee
```
Comment (0x6861746366574) [inline src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts:153]: What is a 'slow fee' defined as?

File: src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts [hunk 2]
```diff
@@ -248,15 +249,25 @@ export class FormPersistentDelegationRequestTransactionTs extends FormTransactio
...
-    private async currentSignerWatch() {
-        this.formItems.signerAddress = this.signerAddress || this.currentSignerAccountInfo?.address.plain();
+    private async currentSignerAccountInfoWatch(newVal: AccountInfo, oldVal: AccountInfo) {
+        if (
+            newVal &&
+            oldVal &&
+            newVal?.publicKey === oldVal?.publicKey &&
+            newVal.supplementalPublicKeys.node?.publicKey === oldVal.supplementalPublicKeys.node?.publicKey
+        ) {
+            // no interesting changes
```
Comment (0x6861746366574) [inline src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts:259]: This comment isn't very descriptive. ;)

File: src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts [hunk 3]
```diff
@@ -248,15 +249,25 @@ export class FormPersistentDelegationRequestTransactionTs extends FormTransactio
     }
 
     @Watch('currentSignerAccountInfo', { immediate: true })
-    private async currentSignerWatch() {
-        this.formItems.signerAddress = this.signerAddress || this.currentSignerAccountInfo?.address.plain();
+    private async currentSignerAccountInfoWatch(newVal: AccountInfo, oldVal: AccountInfo) {
+        if (
+            newVal &&
+            oldVal &&
+            newVal?.publicKey === oldVal?.publicKey &&
```
Comment (0x6861746366574) [inline src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts:256]: Could you walk me through what you are trying to achieve here with this solution? In reference to Issue #1812.

Comment (0x6861746366574) [reply src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts:256]: Nevermind, just read CryptoBeliever's comment. Makes sense. 👍🏻

File: src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts (hunk 2 above)
Comment (yilmazbahadir) [reply src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts:259]: Tbh, I spent some time writing this comment :) And didn't want to repeat what the two lines(256, 257) already say.
Let me find some better words 👍

**Review by AnthonyLaw — APPROVED** [review]
look good mate 👍🏼

File: src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts (hunk 2 above)
Comment (yilmazbahadir) [reply src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts]: @0x6861746366574 Updated the comment.

**Review by 0x6861746366574 — APPROVED** [review]

**Pull request merged**